src/main/scala/uk/gov/homeoffice/drt/api/PublicRoutes.scala

```diff
@@ -0,0 +1,80 @@
+package uk.gov.homeoffice.drt.api
+
+import java.util.concurrent.Executors
+
+import cats.effect._
+import cats.implicits._
+import org.http4s.dsl.Http4sDsl
+import org.http4s.{HttpRoutes, Response, StaticFile, Status}
+import org.slf4j.LoggerFactory
+import uk.gov.homeoffice.drt.AirlineConfig
+import uk.gov.homeoffice.drt.service.AirlineService
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success, Try}
+
+object PublicRoutes {
+  val logger = LoggerFactory.getLogger(getClass.getName)
+
+  val dbExecutionContext = ExecutionContext.global // replace with your DB specific EC.
+
+  val blockingPool = Executors.newFixedThreadPool(4)
+  val blocker = Blocker.liftExecutorService(blockingPool)
```

Comment:
I wonder if we can name these more descriptively as I have no idea what they do

Reply:
Those are http4s variable needs to defined for Static Routing . Don't know what could be better name here .